web/src/forms/settings/IrcForms.tsx

```diff
@@ -27,44 +27,49 @@ interface ChannelsFieldArrayProps {
 }
 
 const ChannelsFieldArray = ({ channels }: ChannelsFieldArrayProps) => (
-  <div className="p-6">
+  <div className="px-4">
     <FieldArray name="channels">
       {({ remove, push }: FieldArrayRenderProps) => (
-        <div className="flex flex-col space-y-2 border-2 border-dashed dark:border-gray-700 p-4">
+        <div className="flex flex-col space-y-2">
           {channels && channels.length > 0 ? (
             channels.map((channel: IrcChannel, index) => {
               const isDisabled = channel.name === "#ptp-announce-dev";
               return (
-                <div key={index} className="flex justify-between">
-                  <div className="flex">
-                    <Field name={`channels.${channel.id}.name`}>
-                      {({ field }: FieldProps) => (
+                <div key={index} className="flex justify-between border dark:border-gray-700 dark:bg-gray-815 p-2 rounded-md">
+                  <div className="flex gap-2">
+                    <Field name={`channels.${index}.name`}>
```

Comment:
Shouldn't this be channel.id?

Reply:
FieldArrays needs the index. It was id before but it did not load the data and errored on change.